src/rhub/api/lab/region.py

```diff
@@ -67,15 +67,16 @@ def list_regions(keycloak: KeycloakClient,
                  user, filter_, sort=None, page=0, limit=DEFAULT_PAGE_LIMIT):
     regions = list_regions_with_permissions(keycloak, user)
 
+    regions = regions.outerjoin(
+        model.Location,
+        model.Location.id == model.Region.location_id,
+    )
+
     if 'name' in filter_:
         regions = regions.filter(model.Region.name.ilike(filter_['name']))
 
     if 'location' in filter_:
-        regions = (
-            regions
-            .outerjoin(model.Location)
-            .filter(model.Location.name.ilike(filter_['location']))
-        )
+        regions = regions .filter(model.Location.name.ilike(filter_['location']))
```

Comment:
Is the space between 'regions' and '.filter' intentional?

Reply:
Baffled that this wasn't caught by the linter